packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1724,10 +1723,6 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
     }
 
     public async getRootDataStore(id: string, wait = true): Promise<IFluidRouter> {
-        return this.getRootDataStoreChannel(id, wait);
-    }
-
-    private async getRootDataStoreChannel(id: string, wait = true): Promise<IFluidDataStoreChannel> {
```

Comment:
this is already a big pr, please revert unnecessary changes

Reply:
i don't think anything in this file needs to change

Reply:
Reverted locally, will come in next push. I'll put it in a PR of its own.